.github/workflows/32-bit-linux.yml

```
@@ -52,3 +52,7 @@ jobs:
           name: Test results
           path: test-data.xml
         if: failure()
+    concurrency:
+      # https://github.community/t/concurrecy-not-work-for-push/183068/7
+      group: ${{ github.event_name == 'push' && github.run_number || github.ref }}
```

```suggestion
      group: ${{ github.event_name == 'push' && github.run_number || github.ref }}-32bit
```

Makes this extra unique just in case another workflow is created (and is the convention with our other jobs)

---

thx